.github/workflows/CI.yml

```
@@ -0,0 +1,49 @@
+    branches: [dev]
+    paths:
+      - '**'
+
+permissions:
+  contents: read
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+    - name: 1. checkout
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)